x/launch/client/cli/tx_request_remove_validator.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+func CmdRequestRemoveValidator() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "request-remove-validator [chainID] [validatorAddress]",
+		Short: "Request to remove a genesis validator",
+		Args:  cobra.RangeArgs(1, 2),
```

Comment:
I like this format👍 I'm wondering if we could have a way to precise `[validatorAddress]` is optional. Normally `[...]` is for optional argument and `<...>` for mandatory one but for some reason Cosmos SDK decided to use `[...]` for everything

Reply:
It will be cool if we have a way to differentiate both... maybe we can use `(...)`, but we are creating a new pattern. I wouldn't say I like it too much.

Reply:
Let's keep it like this for now, this is not problematic if people put the address.

In fact, the CLI will mainly be used for testing purposes